blocks/library/rest-api-latest-posts/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * WordPress dependencies
+ */
+import Placeholder from 'components/placeholder';
+
+/**
+ * Internal dependencies
+ */
+import { registerBlock } from '../../api';
+
+function getLatestPosts( postsToShow = 5 ) {
```

Comment:
Thinking... this could be a good example for breaking some logic into a separate file. Perhaps `rest-api-latest-posts/data.js`?